### PR TITLE
Add intensity to ambient light (fixes #1201)

### LINKED ISSUE #1201: add intensity property to ambient light
**Description:** three.js r74 introduced intensity property on ambient light https://github.com/mrdoob/three.js/pull/7621 . Add that to the light component.

- A-Frame Version: master 
- Platform/Device: all

### PR DESCRIPTION
**Pull request by MichaelRupertDev**
**Description:**
three.js r74 introduced intensity property on ambient light mrdoob/three.js#7621 . Add that to the light component.
**Changes proposed:**
- updated component
- added test

### CODE REVIEW

File: tests/components/light.test.js [hunk 1]
```diff
@@ -38,6 +38,12 @@ suite('light', function () {
     });
   });
 
+  test('can update parameters on updated light', function () {
```
Comment (dmarcos) [inline tests/components/light.test.js]: i would just update the test ```updatesLight``` above to also check for the intensity value

**Comment by dmarcos** [comment]
Almost there!

File: tests/components/light.test.js (hunk 1 above)
Comment (MichaelRupertDev) [inline tests/components/light.test.js]: The only reason I didn't do that is because at that time, it's a directional light, rather than an ambient light.

Comment (dmarcos) [inline tests/components/light.test.js]: This test should be anyways in the update suite and not hanging by itself

**Comment by michaelrupert-wf** [comment]
@dmarcos Updated.

File: tests/components/light.test.js [hunk 2]
```diff
@@ -36,6 +36,12 @@ suite('light', function () {
       el.setAttribute('light', 'type: ambient');
       assert.equal(el.object3D.children[0].type, 'AmbientLight');
     });
+
+    test('can update parameters on updated light', function () {
+      var el = this.el;
+      el.setAttribute('light', 'intensity: 5');
+      assert.shallowDeepEqual(el.object3D.children[0].intensity, 5);
```
Comment (ngokevin) [inline tests/components/light.test.js]: A simple `equal` would do

Comment (dmarcos) [inline tests/components/light.test.js]: Let's replace this with an equal and squash the commits.

**Comment by ngokevin** [comment]
Can you update the docs as well?

File: tests/components/light.test.js (hunk 2 above)
Comment (dmarcos) [inline tests/components/light.test.js]: If we can replace ```shallowDeepEqual``` with ```equal``` I think we can merge

**Comment by MichaelRupertDev** [comment]
@dmarcos @ngokevin Finally fixed the messed up rebase, should be good to go

**Comment by dmarcos** [comment]
Nice! Thank you for keeping at it

**Pull request merged**